org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4viz/AlloyAtom.java

```diff
@@ -40,17 +40,10 @@
      */
     private final int       index;
 
-    /** Create a new AlloyAtom with the given type and index. */
-    public AlloyAtom(AlloyType type, int index) {
-        this.type = type;
-        this.index = index;
-        this.originalName = type.getName() + "." + index;
-    }
-
     /**
      * Create a new AlloyAtom with the given type, index, and label.
      */
-    public AlloyAtom(AlloyType type, int index, String originalName) {
```

Comment:
This is an incompatible change because it can break people outside this codebase.